feat: Implement the modulectl create command (#49)
kyma-project/modulectl: docs/gen-docs/modulectl_create.md

- defaultCR:        a string, optional, reference to a YAML file containing the default CR for the module, must be a relative file name or URL
- mandatory:        a boolean, optional, default=false, indicates whether the module is mandatory to be installed on all clusters
- resourceName:     a string, optional, default={NAME}-{CHANNEL}, the name for the ModuleTemplate CR that will be created
- internal:         a boolean, optional, default=false, determines whether the ModuleTemplate CR should have the internal flag or not
- beta:             a boolean, optional, default=false, determines whether the ModuleTemplate CR should have the beta flag or not
- security:         a string, optional, name of the security scanners config file
- labels:           a map with string keys and values, optional, additional labels for the generated ModuleTemplate CR
- annotations:      a map with string keys and values, optional, additional annotations for the generated ModuleTemplate CR
```

The **manifest** file contains all the module's resources in a single, multi-document YAML file. These resources will be created in the Kyma cluster when the module is activated.

> The **defaultCR** file contains a default custom resource for the module that will be installed along with the module. It will be additionally schema-validated against the Custom Resource Definition.

Reviewer: ```suggestion
The **defaultCR** file contains a default custom resource for the module that will be installed along with the module. It will be additionally schema-validated against the Custom Resource Definition.
```
```suggestion
The **defaultCR** file contains a default custom resource for the module that is installed along with the module. It is additionally schema-validated against the Custom Resource Definition.
```

> The **defaultCR** file contains a default custom resource for the module that will be installed along with the module. It will be additionally schema-validated against the Custom Resource Definition.

Reviewer: ```suggestion
The **defaultCR** file contains a default custom resource for the module that is installed along with the module. It is additionally schema-validated against the Custom Resource Definition.
```
